Add an email to teams about the dispatch of their kits (#124)
srobo/team-emails: SR2023/2022-11-14-kit-dispatch.md

---
to: Student Robotics 2023 teams whose kit has shipped
subject: Kit dispatch
---

Hi,

Good news, your kit is on its way. We appreciate that this has taken longer than
hoped and are grateful for your patience.

> Your kit should arrive within the next few days, you can track its progress at

Reviewer: ```suggestion
Your kit should arrive tomorrow, you can track its progress at
```

We're using overnight shipping, and this batch of kits all got picked up today. UPS are usually pretty good at doing this on time.

Author: :+1: https://github.com/srobo/team-emails/pull/124/commits/29292d2fd50ae1387b37057e2cd714ed2d25edac
